TwitchLib.Api/TwitchAPI.cs

```diff
@@ -35,6 +34,40 @@ public TwitchAPI(ILoggerFactory loggerFactory = null, IRateLimiter rateLimiter =
             V5 = new V5.V5(loggerFactory, _rateLimiter, Settings, _http);
             ThirdParty = new ThirdParty.ThirdParty(Settings, _rateLimiter, _http);
             Undocumented = new Undocumented(Settings, _rateLimiter, _http);
+
+            settings.PropertyChanged += Settings_PropertyChanged;
+        }
+
+        private void Settings_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
```

Comment:
Would change this to SettingsPropertyChanged

Reply:
yes